RevenueCatUI/Templates/LinTemplate5View.swift

```diff
@@ -37,15 +37,16 @@ struct LinTemplate5View: TemplateViewType {
             selectedPackage: $selectedPackage,
             displayImage: true,
             titleProvider: { package in package.localization.title },
-            getDefaultContentWidth: Constants.defaultContentWidth,
+            getDefaultContentWidth: Constants.defaultContentWidth, 
+            horizontalPaddingModifier: DefaultHorizontalPaddingModifier(),
             subtitleBuilder: { EmptyView() },
             buttonSubtitleBuilder: { (_, _ , _) in EmptyView() }
         )
     }
 }
 
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
-struct LinConfigurableTemplate5View<SubtitleView: View, ButtonSubtitleView: View>: View {
+struct LinConfigurableTemplate5View<SubtitleView: View, ButtonSubtitleView: View, HorizontalPadding: ViewModifier>: View {
```

Comment:
so we have this `LinConfigurableTemplate5View`, which is also gets used in `LinConfigurableTemplate4View`.
Now, in 4 we need a slightly different padding for internal labels & that's how we do it.
Normal is unchanged (DefaultHorizontalPaddingModifier), while 4 uses NoPaddingModifier